### DISCUSSION TITLE
캡스톤 디자인 경진대회 최우수상 수상 회고!

### DISCUSSION THREAD

**Discussion by shkisme** [BDD-CLUB/challenge-every-week-posting#342] [5. 회고]
- [캡스톤 디자인 경진대회 최우수상 수상!](https://shkisme.vercel.app/capstone-design-awards)

저번 주에 최우수상을 수상한 두레 프로젝트에 대해 전체적으로 회고해 본 글을 작성해보았습니다!!

**Comment by 02ggang9** [comment]
ㅎ히힛 수고 많으셨습니다~
